subsys/nfc/lib/platform.c

```diff
@@ -29,6 +33,50 @@ LOG_MODULE_REGISTER(nfc_platform, CONFIG_NFC_PLATFORM_LOG_LEVEL);
 
 #define DT_DRV_COMPAT nordic_nrf_clock
 
+/* Number of NFC Tag Header registers in the FICR register space. */
+#define NFC_TAGHEADER_REGISTERS_NUM	3
+
+#ifdef CONFIG_NFC_ZERO_LATENCY_IRQ
+#define NFCT_IRQ_FLAGS	IRQ_ZERO_LATENCY
+#else
+#define NFCT_IRQ_FLAGS	0
+#endif
+
+#ifdef CONFIG_NFC_ZERO_LATENCY_IRQ
+
+#if defined(CONFIG_NFC_SWI0)
+#define NFC_SWI_IRQN		EGU0_IRQn
+#elif defined(CONFIG_NFC_SWI1)
+#define NFC_SWI_IRQN		EGU1_IRQn
+#elif defined(CONFIG_NFC_SWI2)
+#define NFC_SWI_IRQN		EGU2_IRQn
+#elif defined(CONFIG_NFC_SWI3)
+#define NFC_SWI_IRQN		EGU3_IRQn
+#elif defined(CONFIG_NFC_SWI4)
+#define NFC_SWI_IRQN		EGU4_IRQn
+#elif defined(CONFIG_NFC_SWI5)
+#define NFC_SWI_IRQN		EGU5_IRQn
+#else
+#error "NFC software interrupt is undefined."
+#endif
+
+#endif /* CONFIG_NFC_ZERO_LATENCY_IRQ */
+
+#ifdef CONFIG_NFC_THREAD_CALLBACK
+#define MAX_ITEM_SIZE32 CONFIG_NFC_MAX_ITEM_SIZE
+
+static uint32_t item_buf_rx[MAX_ITEM_SIZE32];
+RING_BUF_ITEM_DECLARE_POW2(nfc_cb_ring, CONFIG_NFC_RING_EXP);
+
+static bool buf_full;
+#endif
+
+#ifdef CONFIG_NFC_OWN_THREAD
+K_SEM_DEFINE(cb_sem, 0, 1);
+#endif
+
+static nfc_lib_cb_call_t nfc_cb_call;
```

Comment:
How about modifying the name as below?
```suggestion
static nfc_lib_cb_call_t nfc_cb_resolve;
```
I suggested renaming the type in the relevant PR.

Reply:
In progress right now.